src/com_tjvendors/media/js/tjvendors.js

```diff
@@ -237,7 +237,24 @@ var tjVSite = {
 			Joomla.submitbutton = function(task) {
 				if (task == 'vendor.save') {
 					var validData = document.formvalidator.isValid(document.getElementById('adminForm'));
+					var url = jQuery('#jform_website_address').length ? jQuery('#jform_website_address').val() : '';
+					const urlPattern = new RegExp(
```

Comment:
@Sachintechjoomla this is how client-side validations could be added in joomla

https://manual.joomla.org/docs/general-concepts/forms/client-side-validation/#:~:text=Custom%20Validation%20routine%E2%80%8B&text=Step%201%20Write%20the%20js,false%20if%20it%20isn't.